src/main/webapp/aboutus.html

```diff
@@ -30,11 +30,11 @@
       </ul>
     </nav>
     <h1>About Our Team</h1>
-    <h2>Teammate A Name</h2>
+    <h2>Mia Boloix</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+         <li>Summer Feelz: Very excited, a little overwhelmed, and so ready!!</li>
```

Comment:
I think the indents right now are 5 spaces in this change. let's go back to 2 spaces and do a mass indent fix later.